third_party/2and3/click/termui.pyi

```diff
@@ -59,7 +59,7 @@ def get_terminal_size() -> Tuple[int, int]:
     ...
 
 
-def echo_via_pager(text: str, color: Optional[bool] = ...) -> None:
+def echo_via_pager(text_or_generator: Union[str, Iterable[str]], color: Optional[bool]) -> None:
```

Comment:
The argument can also be a generator function, something like `Callable[[], Generator[str, None, None]]` (https://github.com/pallets/click/blob/master/click/termui.py#L246).

Reply:
oops, missed that at first. Should be good now.

(I am assuming using `Iterable[str]` in place of `Generator[str, None, None]` is fine).

Reply:
I do think it should be Generator, because the check specifically uses `isgeneratorfunction`. `lambda: ["a", "b"]` would not work at runtime.

Reply:
You are right! Updated it now.